### ISSUE TITLE
Missing documentation for DRep and pool voting thresholds

### ISSUE THREAD

**Issue by carlhammann** [input-output-hk/cardano-cli#471]
## Description

Curerently, the documentation on Conway  protocol parameter updates looks like this:

```
  --pool-voting-threshold-motion-no-confidence RATIONAL
                           TODO
  --pool-voting-threshold-committee-normal RATIONAL
                           TODO
  --pool-voting-threshold-committee-no-confidence RATIONAL
                           TODO
  --pool-voting-threshold-hard-fork-initiation RATIONAL
                           TODO
  --drep-voting-threshold-motion-no-confidence RATIONAL
                           TODO
  --drep-voting-threshold-committee-normal RATIONAL
                           TODO
  --drep-voting-threshold-committee-no-confidence RATIONAL
                           TODO
  --drep-voting-threshold-update-to-constitution RATIONAL
                           TODO
  --drep-voting-threshold-hard-fork-initiation RATIONAL
                           TODO
  --drep-voting-threshold-pp-network-group RATIONAL
                           TODO
  --drep-voting-threshold-pp-economic-group RATIONAL
                           TODO
  --drep-voting-threshold-pp-technical-group RATIONAL
                           TODO
  --drep-voting-threshold-pp-governance-group RATIONAL
                           TODO
  --drep-voting-threshold-treasury-withdrawal RATIONAL
                           TODO
  --min-committee-size INT TODO
  --committee-term-length INT
                           TODO
  --governance-action-lifetime NATURAL
                           TODO
```

**Comment by carlhammann** [comment]
Thanks to @james-iohk for finding this one!

**Comment by carlhammann** [comment]
This is included in #482, so I'll close this.
